spec/rubocop/cop/style/hash_syntax_spec.rb

```diff
@@ -1268,4 +1268,76 @@ def do_something
       end
     end
   end
+
+  context 'configured to disallow mixing of implicit and explicit hash literal value' do
+    let(:cop_config) do
+      {
+        'EnforcedStyle' => 'ruby19',
+        'SupportedStyles' => %w[ruby19 hash_rockets],
+        'EnforcedShorthandSyntax' => 'consistent'
+      }
+    end
+
+    context 'Ruby >= 3.1', :ruby31 do
+      it 'does not register an offense when all hash values are omitted' do
+        expect_no_offenses(<<~RUBY)
+          {foo:, bar:}
+        RUBY
+      end
+
+      it 'registers an offense when some hash values are omitted but they can all be omitted' do
+        expect_offense(<<~RUBY)
+          {foo:, bar: bar}
+                      ^^^ Do not mix explicit and implicit hash values.  Omit the hash value.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          {foo:, bar:}
+        RUBY
+      end
+
+      it 'registers an offense when some hash values are omitted but they cannot all be omitted' do
+        expect_offense(<<~RUBY)
+          {foo:, bar: baz}
+           ^^^ Do not mix explicit and implicit hash values.  Explicit the hash value.
```

Comment:
`Explicit the hash value` doesn't really make sense.
```suggestion
           ^^^ Do not mix explicit and implicit hash values.  Include the hash value.
```

Reply:
I agree that it doesn't make much sense, but I was reusing the existing error message for the `never` option.  Should I change that one too - are changes to existing error messages a breaking change?  

We can agree to disagree on the period double space formatting though 😉.  I will remove it though as it does seem more consistent with the rest of the error messages.

Reply:
We could change it as a separate PR if you'd prefer, but if you're willing, change it everywhere please! Changing messages aren't considered breaking.

Reply:
Added a commit to reword the existing message before adding the new feature.